### DISCUSSION TITLE
STTextView height can not be smaller than the window - STLineNumberRulerView bug

### DISCUSSION THREAD

**Discussion by mefuzar** [krzyzanowskim/STTextView#44] [Questions]
Hello,
I would like to embed STTextView in my view but when settings top and bottom constraints which are different than zero (0), it will draw the line numbers outside of the frame.

Please see attached image for demonstration.

How to reproduce?
1. Open TextEdit example Project.
2. Change lines 51 & 52 from:
```swift
            scrollView.topAnchor.constraint(equalTo: view.topAnchor),
            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
``` 
to:
```swift
            scrollView.topAnchor.constraint(equalTo: view.topAnchor, constant: 100.0),
            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -100.0)
``` 
![Screenshot 2024-01-23 at 14 09 24](https://github.com/krzyzanowskim/STTextView/assets/85771020/9ed00b8d-77d5-4343-92ea-627f04268300)

Please see attached,  note numbers 75 to 79 / 97+ besides the frame

Thank you in advance

**Comment by krzyzanowskim** [comment]
The line number ruler is part of the NSScrollView container (`NSScrollView.verticalRulerView`), so I think you need to set the frame for the NSScrollView
